acestep/audio_utils.py

```
@@ -616,3 +618,60 @@ def save_audio(
+        logger.warning(f"[Tags] Audio file not found, skipping tags: {audio_path}")
+        return
+
+    tmp = src.with_suffix(f".tmp{src.suffix}")
+    try:
+        cmd = [ffmpeg, '-y', '-hide_banner', '-loglevel', 'error', '-i', str(src), '-c', 'copy']
+        for key, value in clean_tags.items():
+            # Escape any double-quotes inside the value so the shell doesn't break
+            cmd += ['-metadata', f'{key}={value}']
+        cmd.append(str(tmp))
+
+        subprocess.run(cmd, check=True, capture_output=True, timeout=60)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)